ansible/istio/defaults/main.yml

```diff
@@ -29,4 +25,7 @@ istio:
   bookinfo: false
 
   # Whether or not to open apps in the browser
-  open_apps: false
\ No newline at end of file
+  open_apps: false
+
+  # Whether to delete the old namespace and all data if it exists
+  delete_old: false
```

Comment:
Why do you name the param using delete_old ? Such param should be used to delete istio according to namespace param ....

Reply:
This is just a flag that controls whether deleting will take place or not.
If this flag is set to true, then all the stuff under `istio.namespace` will be deleted.
If the flag is not set then no delete operations will take place before deploying Istio

Reply:
My question was more related to "_old". Why do you suffix the name with "_old" ?